ci-operator/step-registry/agent/post/agent-post-chain.yaml

```diff
@@ -2,6 +2,7 @@ chain:
   as: agent-post
   steps:
   - ref: baremetalds-devscripts-gather
+  - ref: agent-devscripts-gather
```

Comment:
About the naming, I'd personally to prefer a simple `agent-gather` (using dev-scripts here it's a tech implementation details, and in future it could change. The scope of the step it's anyhow to gather logs relevant for the agent case)